### PR TITLE
Ability to add arbitrary Helm charts with Pipeline ...

### PR DESCRIPTION
**Pull request by baluchicken**
besides the predefined `spotguides` one.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -538,8 +532,8 @@ func updatePrometheusPostHook(_ *banzaiSimpleTypes.ClusterSimple, _ *gin.Context
 
 func installHelmPostHook(createdCluster *banzaiSimpleTypes.ClusterSimple, c *gin.Context) {
 	logTag := "InstallHelmPostHook"
-	retryAttempts := viper.GetInt("dev.retryAttempt")
-	retrySleepSeconds := viper.GetInt("dev.retrySleepSeconds")
+	retryAttempts := viper.GetInt("helm.retryAttempt")
```
Comment (stoader) [inline main.go]: `helm.retryAttempt` and `helm.retrySleepSeconds` should be constant.

Comment (matyix) [reply main.go]: I believe I've asked to make configurable due to some AKS issues, but constants are fine as well,

Comment (matyix) [reply main.go]: @baluchicken how about dealing with the goroutines (for all hooks)? Will open a Jira ticket for that.

Comment (stoader) [reply main.go]: I meant to the `helm.retryAttempt` literal to be a constant like `const HELM_RETRY_ATTEMPT_CONFIG="helm.retryAttempt"`

**Pull request merged**